### PR TITLE
feat: SfStorelocator map centering  on click on shop card

### LINKED ISSUE #1281: [FEATURE] SfStoreLocator: clicking anywhere on the store card should centre the store on the map
Currently, to center the store on the map one needs to click on the picture of the store.

We should alter this behavior and either have a dedicated button or have the whole store card as clickable.

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue

Closes #1281

# Scope of work
SfStoreLocator - SfButton added to store card and click event centering map on store. 
Adjust styling to current designs.  

# Screenshots of visual changes
image

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package-lock.json
 .DS_Store
 node_modules
+vsf-test
```
Comment (justyna-13) [inline .gitignore:4]: Why is it here? :)

File: packages/shared/styles/components/organisms/SfStoreLocator.scss [hunk 2]
```diff
@@ -12,28 +12,48 @@
     color: var(--c-dark-variant);
     @include font(
       --store-item-distance-font,
-      var(--font-light),
+      var(--font-normal),
```
Comment (justyna-13) [inline packages/shared/styles/components/organisms/SfStoreLocator.scss:15]: Would be good to update branch with develop and change all on the current vars, wdyt? :)

File: .gitignore (hunk 1 above)
Comment (AdamPawlinski) [reply .gitignore:4]: I added test project in packages by mistake and then tried to remove it, but it shouldn't be here. Thanks for let me know :+1:

File: packages/shared/styles/components/organisms/SfStoreLocator.scss (hunk 2 above)
Comment (AdamPawlinski) [reply packages/shared/styles/components/organisms/SfStoreLocator.scss]: Of course, you are right. I fixed that.

File: packages/shared/styles/components/organisms/SfStoreLocator.scss [hunk 3]
```diff
@@ -12,39 +12,59 @@
...
+      var(--font-family--secondary)
     );
     span {
-      font-weight: var(--font-bold);
+      font-weight: var(--font-weight-bold);
       color: var(--c-text);
     }
   }
+  &__btn {
+    --button-background: var(--c-white);
```
Comment (justyna-13) [inline packages/shared/styles/components/organisms/SfStoreLocator.scss:26]: add to SfButton class --pure modifier and then you can remove most properties in btn and btn:active

**Review by justyna-13 — CHANGES_REQUESTED** [review]

File: packages/shared/styles/components/organisms/SfStoreLocator.scss (hunk 3 above)
Comment (AdamPawlinski) [reply packages/shared/styles/components/organisms/SfStoreLocator.scss]: Thanks, great idea :)

**Review by filrak — APPROVED** [review]

**Pull request merged**